Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:18.04
 
 MAINTAINER Zostera B.V.
-LABEL version="0.2.4"
+LABEL version="0.2.5"
```

Comment:
I'd say 0.3.0, it's not a bugfix but a significant improvement.

Reply:
Done